less/project/theme-extras.less

```diff
@@ -196,6 +196,52 @@
 
 }
 
+// --------------------------------------------------
+// Component
+// --------------------------------------------------
+
+// Component header bg color mixin
+// Can be added to component to enable
+// e.g. 'comp-header-bg-color black'
+// Note: both arguments must be predefined variables
+.component-header-bg-mixin(black, white);
+
+.component-header-bg-mixin(@color, @color-inverted) {
+
+.comp-header-bg-color.@{color} {
+  .component {
+    &__inner {
+      margin: -@component-padding-ver -@component-padding-hoz;
+    }
+
+    &__header {
+      padding: @component-padding-ver @component-padding-ver;
+      background-color: @@color;
+      color: @@color-inverted;
+    }
+
+    &__title,
+    &__body a,
+    &__instruction a {
+      color: @@color-inverted;
+    }
+  }
+
+  .pagelevelprogress {
+    &__indicator {
+      border-color: @@color-inverted;
+    }
+    &__indicator-inner {
+      background-color: @@color;
+    }
+    &__indicator-bar {
+      background-color: @@color-inverted;
+    }
```

Comment:
```suggestion
    &__indicator {
      border-color: @@color-inverted;
      
      &-bar {
        background-color: @@color-inverted;
      }
    
      &-inner {
        background-color: @@color;
      }
    }
```

Reply:
Are you happy to leave this as is as it follows the `bg-color` definition in Vanilla's `theme-extras` https://github.com/adaptlearning/adapt-contrib-vanilla/blob/master/less/project/theme-extras.less#L228

Reply:
yes of course no problem